src/acceptance-test/java/net/consensys/orion/acceptance/MultiKeyOperationAcceptanceTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.orion.acceptance;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import net.consensys.orion.acceptance.dsl.AcceptanceTestBase;
+import net.consensys.orion.acceptance.dsl.OrionInstance;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import org.junit.Test;
+
+public class MultiKeyOperationAcceptanceTest extends AcceptanceTestBase {
+  private final byte[] payload = "This Is My Data".getBytes(StandardCharsets.UTF_8);
+
+  @Test
+  public void dataTransfersFromOneNodeToAnother() throws IOException, InterruptedException {
+    final OrionInstance bootnode = orionFactory().create("node_1", 1);
+    bootnode.start();
+
+    final OrionInstance secondNode = orionFactory().create("node_2", 1, singletonList(bootnode));
+    secondNode.start();
+
+    Thread.sleep(1000);
```

Comment:
Nasty, is something being waited on here? ...or why are we waiting 1000ms?

Reply:
This IS HORRIBLE - if you have a good answer - love to hear.
Basically - you need to wait until the second node connects onto the bootnode before you do a transfer .. or else it doesn't work.
ATM - orion has no way of reporting connectivity ... maybe we need to _do_ a transfer to prove connectivity is up?

Reply:
Hmmm.... something like getPeerCount() is needed then?

Reply:
that MAY break security - i.e. is Orion to report who it can talk with?